src/components/FDC3Tester/src/Context.jsx

```diff
@@ -0,0 +1,42 @@
+
+import React from 'react'
+import ApiExample from './components/ApiExample'
+
+
+export default function Context() {
+  return (
+    <div>
+      {/* broadcast */}
+      <ApiExample
+        apiName="broadcast"
+        title="broadcast"
+        description="Broadcast Context."
+        codeAction={context => fdc3.broadcast(context)}
+        snippet={(context) => `fdc3.broadcast(${context})`}
+        inputLabel="broadcast:"
+        placeholder='{"type":"fdc3.instrument","id":{"ticker":"AAPL"}}'
+      />
+
+      {/* addContextListener */}
+      <ApiExample
+        apiName="addContextListener"
+        title="addContextListener"
+        description="Adds a listener for incoming Context."
+        codeAction={(...args) => fdc3.addContextListener(...args)}
```

Comment:
It isn't clear to me how to test `addContextListener`. I'm not sure what is supposed to happen when the listener is triggered for these tests. It should print the context to the output panel, but I don't think it does that.